### ISSUE TITLE
Dynamically Assigning Adafruit IO credentials to Adafruit MQTT  object

### ISSUE THREAD

**Issue by Pius171** [adafruit/Adafruit_MQTT_Library#202]
- Arduino board:  **ESP8266**

- Arduino IDE version (found in Arduino -> About Arduino menu):  **1.8.16**

- List the steps to reproduce the problem below (if possible attach a sketch or
  copy the sketch code in too): **LIST REPRO STEPS BELOW**
I want to be able to dynamically assign my MQTT credentials. I want to store them in a file then read them in the setup function and assign them to my Adafruit MQTT object. I have looked through the library files but I can't find a way to do it. 

In short, I want to be able to change my Adafruit io credentials without having to change them manually from the code.

Thank you.
